src/Riok.Mapperly/Configuration/MergedMapperConfiguration.cs

```diff
@@ -0,0 +1,104 @@
+using Microsoft.CodeAnalysis;
+using Riok.Mapperly.Abstractions;
+using Riok.Mapperly.Descriptors;
+using Riok.Mapperly.Helpers;
+
+namespace Riok.Mapperly.Configuration;
+
+public class MergedMapperConfiguration
```

Comment:
Wouldn't be a name like `MapperConfigurationReader` more appropriate here?

Reply:
Changed to `MapperConfigurationReader`.